data/json/monsters/reptile_amphibian.json

```diff
@@ -361,7 +363,7 @@
     "vision_night": 35,
     "harvest": "mutant_animal_large_noskin",
     "dissect": "dissect_lizard_sample_large",
-    "//grab": "Pretty small, considering the description, but multiple teeth sound grabby",
+    "//grab": "Pretty small, considering the description, but multiple teeth sound grabby - outdated, now it's a proper big nasty rope",
```

Comment:
Instead of adding a note to a note saying the note is outdated why not either update the note or remove it entirely?

Reply:
Updated the note